src/flutter/flutter-wizard.ts

```diff
@@ -0,0 +1,119 @@
+import { WizardOptions } from '../utils/types';
+import * as Sentry from '@sentry/node';
+import * as codetools from './code-tools';
+import * as fs from 'fs';
+
+// @ts-ignore - clack is ESM and TS complains about that. It works though
+import * as clack from '@clack/prompts';
+import chalk from 'chalk';
+
+import {
+  confirmContinueIfNoOrDirtyGitRepo,
+  getOrAskForProjectData,
+  printWelcome,
+} from '../utils/clack-utils';
+
+import { traceStep, withTelemetry } from '../telemetry';
+import { findFile } from './code-tools';
+
+export async function runFlutterWizard(options: WizardOptions): Promise<void> {
+  return withTelemetry(
+    {
+      enabled: options.telemetryEnabled,
+      integration: 'android',
+      wizardOptions: options,
+    },
+    () => runFlutterWizardWithTelemetry(options),
+  );
+}
+
+async function runFlutterWizardWithTelemetry(
+  options: WizardOptions,
+): Promise<void> {
+  printWelcome({
+    wizardName: 'Sentry Flutter Wizard',
+    promoCode: options.promoCode,
+  });
+
+  await confirmContinueIfNoOrDirtyGitRepo();
+
+  const { selectedProject, selfHosted, sentryUrl, authToken } =
+    await getOrAskForProjectData(options, 'flutter');
+
+  const projectDir = process.cwd();
+  const pubspecFile = findFile(projectDir, 'pubspec.yaml');
+
+  // ======== STEP 1. Add sentry_flutter and sentry_dart_plugin to pubspec.yaml ============
+  clack.log.step(
+    `Adding ${chalk.bold('Sentry')} to your apps ${chalk.cyan(
+      'pubspec.yaml',
+    )} file.`,
+  );
+  const pubspecPatched = await traceStep('Patch pubspec.yaml', () =>
+    codetools.patchPubspec(
+      pubspecFile,
+      selectedProject.slug,
+      selectedProject.organization.slug,
+    ),
+  );
+  if (!pubspecPatched) {
+    clack.log.warn(
+      "Could not add Sentry to your apps pubspec.yaml file. You'll have to add it manually.\nPlease follow the instructions at https://docs.sentry.io/platforms/flutter/#install",
+    );
+  }
+  Sentry.setTag('pubspec-patched', pubspecPatched);
+
+  // ======== STEP 2. Add sentry.properties with auth token ============
+
+  const propertiesAdded = traceStep('Add sentry.properties', () =>
+    codetools.addProperties(pubspecFile, authToken),
+  );
+  if (!propertiesAdded) {
+    clack.log.warn(
+      `We could not add "sentry.properties" file in your project directory in order to provide an auth token for Sentry CLI. You'll have to add it manually, or you can set the SENTRY_AUTH_TOKEN environment variable instead. See https://docs.sentry.io/cli/configuration/#auth-token for more information.`,
```

Comment:
Usually filenames are highlighted by `cyan`.

Wizard Develop docs describe the use of colors.

https://develop.sentry.dev/sdk/expected-features/setup-wizards/#colors